esp/esp/program/models/__init__.py

```diff
@@ -1151,38 +1151,95 @@ def getVolunteerRequests(self):
 
     @cache_function
     def getShirtInfo(self):
-        shirt_count = defaultdict(lambda: defaultdict(int))
+        shirts = []
+        shirt_types = [x.strip() for x in Tag.getTag('shirt_types', default = 'Straight cut, Fitted cut').split(',')]
+
         teacher_dict = self.teachers()
-        if 'class_approved' in teacher_dict:
-            query = teacher_dict['class_approved']
-            query = query.filter(registrationprofile__most_recent_profile=True)
-            if not Tag.getBooleanTag('teacherinfo_shirt_type_selection', default=True):
-                query = query.values_list('registrationprofile__teacher_info__shirt_size')
-                query = query.annotate(people=Count('id', distinct=True))
+        teacher_types = {'Approved Teachers': 'class_approved', 'All Teachers': 'class_submitted'}
+        shirt_sizes = [x.strip() for x in Tag.getTag('teacher_shirt_sizes', default = 'XS, S, M, L, XL, XXL').split(',')]
```

Comment:
Now that there is a global tagdict with the default values for each tag, is there a way to write this that doesn't involve specifying the default value?

Reply:
My plan is to make a separate change to `getTag` for that in a separate PR ;)

Reply:
ok, sounds good.